package.json

```
@@ -43,8 +43,9 @@
     "lint:css": "npm run prettier -- --check",
     "prettier": "prettier \"**/*.css\"",
     "postinstall": "npm run bootstrap",
-    "seed": "npm-run-all -p seed:*",
+    "seed": "npm-run-all -p seed:auth-user",
     "seed:auth-user": "cross-env DEBUG=fcc:* node ./tools/scripts/seed/seedAuthUser",
```

```suggestion
    "seed": "cross-env DEBUG=fcc:* node ./tools/scripts/seed/seedAuthUser",
```
Might as well simplify this, since we're running one seed command at once.

---

I assumed this was like this to prepare for the case where we seed multiple users/data. But, I suppose it is easily undone